test/OpenTelemetry.Tests/Trace/Propagation/B3PropagatorTest.cs

```diff
@@ -103,6 +103,28 @@ public void ParseZeroSampled()
             Assert.Equal(new PropagationContext(activityContext, default), this.b3propagator.Extract(default, headersNotSampled, Getter));
         }
 
+        [Fact]
+        public void ParseTrueSampled()
```

Comment:
We could change from Fact to Theory, and pass two inlineData, one using "1" and other using "true".